### PR TITLE
Differential

### PR DESCRIPTION
**Pull request by szarnyasg**
Do not merge yet.

### CODE REVIEW

File: scripts/run.py [hunk 1]
```diff
@@ -88,13 +88,13 @@ def visualize():
     Visualizes the benchmark results
     """
     clean_dir("diagrams")
-    set_working_directory("reporting")
```
Comment (georghinkel) [inline scripts/run.py]: I haven't tried the reporting you built there, but are you sure that all requirements are also met by this reporting tool? I will test it today in the evening, but I would say we could still merge the PR and if really necessary add the old reporting as a second option.

**Comment by szarnyasg** [comment]
@georghinkel sorry I did not indicate clearly that these are WIP PRs. Will let you know once they are ready to merge.

**Comment by georghinkel** [comment]
All fine, I am curious to try it out, anyways

**Comment by agarciadom** [comment]
Oddly enough, the "-e" option does not seem to be there in `run.py` anymore. How come? I checked in `master` and we still have it.

This is a WIP, but please make sure Travis is happy before getting anything merged. I'm happy to tweak the Travis checks if needed.

**Comment by szarnyasg** [comment]
@bluezio good point, see #45 which should fix CI.

**Comment by agarciadom** [comment]
Can you rebase this PR on top of the latest master, which has #45 merged into it?

**Comment by szarnyasg** [comment]
@bluezio sure, done - CI's now green.

**Comment by szarnyasg** [comment]
This is good to merge now.

**Comment by agarciadom** [comment]
This seems to be missing the changes to the Dockerfile. Could you update it?

**Comment by szarnyasg** [comment]
@bluezio do you mean the CI?
Come to think of it, we could easily run a Docker image in Travis (which is what we do in most [LDBC](https://github.com/ldbc/ldbc_snb_datagen/blob/master/.travis.yml) [repos](https://github.com/ldbc/ldbc_snb_docs/blob/master/.travis.yml)).

**Comment by agarciadom** [comment]
Oh sorry, got confused with the TTC 2019 cases which do have `Dockerfile`s.

I think we can merge the current version for now. If you are able to create a Dockerfile that can span all these pieces of software, it'd be great, but I do realize that would be quite an ordeal!

**Pull request merged**